eng/common/cross/build-rootfs.sh

```diff
@@ -56,6 +56,10 @@ __UbuntuPackages+=" libkrb5-dev"
 __UbuntuPackages+=" libssl-dev"
 __UbuntuPackages+=" zlib1g-dev"
 
+# ML.NET dependencies
```

Comment:
Since this is in `eng\common`, should this be changed in `dotnet/arcade`?

Reply:
I have already made this change in `dotnet/arcade` and the PR is merged, but the latest version of arcade breaks the ML.NET build. While I am troubleshooting the issue I have made the change locally so we can still take advantage of it.